addons/property_panel/properties.gd

```diff
@@ -136,17 +136,17 @@ class RangeProperty extends Property:
 
 
 class IntProperty extends RangeProperty:
-	func _init(_name,_from,_to,_default = _from):
+	func _init(_name,_from=0,_to=100,_default = _from,_step = _from):
```

Comment:
I don't think `_from` is a good default for `_step` here, 1 would make more sense.